# The incoming command message from coalition leader

uint64 sendingTime #The sending time

uint16 cmdTypeID

string cmdMessage
